create_dataset/create_dataset.py

```diff
@@ -0,0 +1,133 @@
+"""Create dataset of burrowing owl vocalizations and noise.
+
+This script will parse through 2017 and 2018 human labeled
+burrowing owl data. It will create a folder with segments of
+labeled detections, and an equal number of noise samples from
+the same wav files. It will create a CSV with metadata associated
+with the segments. The metadata will include the UUID of the segment,
+the label, the original filepath of the original wav the segment came
+from, the path to the segment, and the start and end time of the
+labeled detection relative to the original wav file. The labeled
+segments will be the duration of the label, and the duration of the
+noise will be fixed and consistent. The user of the dataset may choose
+to pad the labeled detections if they need consistent length segments.
+
+Usage:
+    python3 create_dataset.py -labels /path/to/human/labeled.csv
+    -wav_dir /path/to/parent/dir/of/wavs/
+    -output_dir /path/to/desired/output/dir/
+    -class_list /path/to/classes.txt
+
+"""
+import argparse
+import ntpath
+import os
+import pandas as pd
+from create_segments import get_paths, create_segments
+from create_segments import create_noise_segments
+from filter_labels import filter_labels_2017, filter_labels_2018
+
+
+def create_dataset(labels, wav_dir, output_dir, class_list):
+    """Creates labeled and non labeled segments and metadata.
+
+    Creates segments based on human labeled data of a detection,
+    and then creates an equal number of randomized 'non-detection'
+    segments at fixed length. It cretaes a uuid for each segment
+    and spits out a metadata file that matches the segment to its
+    label, original wav file, relative start time to original wav,
+    and duration.
+
+    Args:
+        labels (str): Path to label file.
+        wav_dir (str): Path to original wav segments of audio.
+        output_dir (str): Path to where the segments and metadata
+            will go.
+        class_list (str): Path to file containing the classes
```

Comment:
It doesn't block approval for me, but documentation on what format the labels file and class_list file are in might be useful (JSON, CSV, column names exist, etc) in case folks want to use/modify in the future. Doesn't block approval for me since the script is meant to create the buow dataset, not any audio dataset